csm/cli/support_bundle/support_bundle.py

```diff
@@ -189,14 +190,23 @@ async def bundle_status(command):
         :param command: Csm_cli Command Object :type: command
         :return: None
         """
-        bundle_id = command.options.get("bundle_id")
-        conf = GeneralConfig(Yaml(const.DATABASE_CLI_CONF).load())
-        db = DataBaseProvider(conf)
-        repo = SupportBundleRepository(db)
-        all_nodes_status = await repo.retrieve_all(bundle_id)
-        response = {"status": [each_status.to_primitive() for each_status in
+        try:
+            bundle_id = command.options.get("bundle_id", "")
+            conf = GeneralConfig(Yaml(const.DATABASE_CONF).load())
+            db = DataBaseProvider(conf)
+            repo = SupportBundleRepository(db)
+            all_nodes_status = await repo.retrieve_all(bundle_id)
+            response = {"status": [each_status.to_primitive() for each_status in
                                all_nodes_status]}
-        return Response(output = response, rc = CSM_OPERATION_SUCESSFUL)
+            return Response(output = response, rc = CSM_OPERATION_SUCESSFUL)
+        except DataAccessExternalError as e:
+            Log.error(f"Failed to connect to elasticsearch: {e}")
+            return Response(output = "Bundle status failed to connect to elasticsearch.",
+                            rc = str(errno.ECONNREFUSED))
+        except Exception as e:
+            Log.error(f"Failed to get bundle status: {e}")
+            return Response(output = "Failed  to get status of bundle.",
```

Comment:
Minor: two spaces between "Failed" and "to".

Reply:
corrected